config/translations/id/generate.controller.yml

```diff
@@ -1,22 +1,22 @@
-description: 'Generate & Register a controller'
-help: 'The <info>generate:controller</info> command helps you generate a new controller.'
-welcome: 'Welcome to the Drupal Controller generator'
+description: 'Buat dan daftarkan controller'
+help: 'Perintah <info>generate:controller</info> membantu kamu untuk membuat controller baru.'
+welcome: 'Selamat datang ke pembuatan Controller Drupal'
 options:
     module: common.options.module
-    class: 'Controller Class name'
-    routes: 'The routes, must be an array containing [title, method, path]'
+    class: 'Nama Kelas Controller'
+    routes: 'Semua route, harus berupa array yang berisi [nama, metode, path]'
     services: common.options.services
-    test: 'Generate a test class'
+    test: 'Buat kelas untuk unit test'
 questions:
     module: common.questions.module
-    class: 'Enter the Controller class name'
-    title: 'Enter the Controller method title (leave empty and press enter when done)'
-    method: 'Enter the action method name'
-    path: 'Enter the route path'
+    class: 'Masukkan nama kelas Controller'
+    title: 'Masukkan judul metode Controller (jika selesai, biarkan kosong dan tekan enter)'
+    method: 'Masukkan nama metode'
+    path: 'Masukkan path untuk route'
     services: common.questions.services
-    test: 'Do you want to generate a unit test class'
+    test: 'Apakah kamu mau membuat kelas untuk unit test?'
```

Comment:
Lol. Feel free to fix it. How about hendak? :-P

On Thu, May 12, 2016, 5:08 PM Ardi <notifications@github.com> wrote:

> In config/translations/id/generate.controller.yml
> <https://github.com/hechoendrupal/DrupalConsole/pull/2245#discussion_r62987204>
> :
>
> >      services: common.questions.services
> > -    test: 'Do you want to generate a unit test class'
> > +    test: 'Apakah kamu mau membuat kelas untuk unit test?'
>
> @loziju <https://github.com/loziju> I think it sounds better to use ingin
> instead of mau, but for now I think is good enough.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/hechoendrupal/DrupalConsole/pull/2245/files/338b9bdc5ac0e7d1cc36724a89410dce428d807b#r62987204>
>